book/src/volume-cloning.md

```diff
@@ -5,7 +5,8 @@
 Status | Min k8s Version | Max k8s version | external-provisioner Version
 --|--|--|--
 Alpha | 1.15 | 1.15 | 1.3
-Beta | 1.16 | - | 1.4
+Beta | 1.16 | 1.17 | 1.4
+GA   | 1.18 | - | 1.4
```

Comment:
We marked cloning as GA in provisioner v1.6.0, because it has a couple of fixes and the finalizer thing.